src/components/SideBySide.js

```diff
@@ -9,13 +9,27 @@ import { isMdxType } from '../utils/mdx';
 const SideBySide = ({ className, children, type }) => {
   const types = Array.isArray(type) ? type : [type];
   const childObjects = Children.toArray(children);
+
+  const isGatsbyMdxType = (child, type) => {
+    if (type === 'img' && isMdxType(child, 'span')) {
+      if (Array.isArray(child.props.children)) {
+        return (
+          child.props.children.filter((child) => isMdxType(child, 'img'))
+            .length > 0
```

Comment:
```suggestion
          child.props.children.some((child) => isMdxType(child, 'img'))
```

Rather than filtering and then checking length, you can use `.some` which will return `true` if at least one of the items is `true`

Reply:
Oh nice, that's handy